### PR TITLE
Spec updates for new bounding volumes

### LINKED ISSUE #48: Update tiles.json bounding volume properties
When bounding volumes for 3D tiles has been finalized, update the tiles.json specification, examples and descriptions.

These are in at least the following places:
[3D Tiles: Tiles.json](https://github.com/AnalyticalGraphicsInc/3d-tiles#tilesjson)
[3D Tiles: External Tilesets](https://github.com/AnalyticalGraphicsInc/3d-tiles#external-tilesets)
[Example tiles.json](https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/master/examples/tiles.json)
[Tiles.json schema](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/schema)

The work on bounding volumes is detailed in [Issue #10 Different bounding volumes](https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/10).

### PR DESCRIPTION
**Pull request by pjcozzi**
Fixes #48 and #10

We'll consider AABB and k-DOPs (both originally discussed in #10) if/when needed.

Cesium implementation: https://github.com/AnalyticalGraphicsInc/cesium/pull/3325

@e-andersson and @lilleyse please review.

### CODE REVIEW

**Comment by pjcozzi** [comment]
This is ready.  Any feedback?

File: schema/boundingVolume.schema.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "$schema" : "http://json-schema.org/draft-03/schema",
+    "title" : "Tile",
+    "type" : "object",
+    "description" : "A bounding volume that encloses a tile or its contents.  At least one property is required.  If more than one property is defined, the runtime can determine which to use.",
+    "properties" : {
+        "box" : {
+            "type" : "array",
+            "description" : "An array of 12 numbers that define a oriented bounding box in WGS84 coordinates.  The first three elements define the x, y, and z values for the center of the box.  The next three elements (with indices 3, 4, and 5) define the x axis direction and length.  The next three elements (indices 6, 7, and 8) define the y axis direction and length.  The last three elements (indices 9, 10, and 11) define the z axis direction and length.",
```
Comment (e-andersson) [inline schema/boundingVolume.schema.json]: Not quite sure I understand this. The first three numbers is the center vector, and the following groups of three numbers are vectors that specify lengths and directions of the box's sides, is that right? Does `axis` here refer to the box's own axes?

**Comment by e-andersson** [comment]
Everything else looks clear.

File: schema/boundingVolume.schema.json (hunk 1 above)
Comment (pjcozzi) [inline schema/boundingVolume.schema.json]: Yes and yes.  This is a standard OBB representation.

Comment (e-andersson) [inline schema/boundingVolume.schema.json]: Ok.

**Comment by lilleyse** [comment]
I made some small edits. Looks good to me.

**Pull request merged**